selenium/src/main/scala/not/ogame/bots/selenium/SeleniumOgameDriverCreator.scala

```diff
@@ -1,24 +1,13 @@
 package not.ogame.bots.selenium
 
-import cats.effect.{Resource, Sync, Timer}
+import cats.effect.{Sync, Timer}
 import not.ogame.bots.{Credentials, LocalClock, OgameDriver, OgameDriverCreator}
-import org.openqa.selenium.firefox.{FirefoxDriver, FirefoxOptions}
+import org.openqa.selenium.WebDriver
 
-class SeleniumOgameDriverCreator[F[_]: Sync](urlProvider: UrlProvider, options: FirefoxOptions = new FirefoxOptions())(
-    implicit timer: Timer[F],
-    clock: LocalClock
-) extends OgameDriverCreator[F] {
-  override def create(credentials: Credentials): Resource[F, OgameDriver[F]] = {
-    Resource
-      .make(Sync[F].delay(createFirefoxDriver()))(r => Sync[F].delay(r.close()))
-      .map { implicit driver =>
-        new SeleniumOgameDriver(credentials, urlProvider)
-      }
-  }
-
-  private def createFirefoxDriver(): FirefoxDriver = {
-    val driver = new FirefoxDriver(options)
-    driver.manage().window().maximize()
-    driver
+class SeleniumOgameDriverCreator[F[_]: Sync](webDriver: WebDriver)(implicit timer: Timer[F], clock: LocalClock)
```

Comment:
You cannot use one WebDriver for two credentials so it should not be here. We can fix that later. I'm thinking about removing this class altogether. It brings lots of problems and isn't doing much anyway.

Reply:
Very good point, we can't. And yes right now when we moved resource acquisition out of it it doesn't do much.

Reply:
But if we remove it we will end up with the previous problem of duplicated credentials ;)

Reply:
Also we can't move webDriver to `create` method because the interface is defined within the core module

Reply:
So might be we should move `credentials` to constructor?